charts/stable/traefik/values.yaml

```diff
@@ -139,7 +139,6 @@ service:
     ports:
       main:
         port: 9000
-        targetPort: 9000
```

Comment:
Actually, no... in accordance to what Traefik publishes it should stay on 9000 internally.